tests/udt/src/lib.rs

```diff
@@ -18,7 +18,8 @@ pub struct UdtStruct {
 
 pub struct Contract;
 
-#[contractimpl(export_if = "export")]
+#[cfg_attr(feature = "export", contractimpl(export = true))]
+#[cfg_attr(not(feature = "export"), contractimpl)]
```

Comment:
Fyi @jonjove I took a quick stab of an alternative approach that shifts the conditional decision on whether to export out of the contractimpl and into user code. It's pretty verbose though. This doesn't actually address any of the issues from #408, it was just an experiment in an alternative way to provide the feature selection.